.github/workflows/e2e-full.yml

```diff
@@ -33,10 +33,9 @@ jobs:
           # https://submariner.io/development/building-testing/ci-maintenance/
           - k8s_version: '1.19'
```

Comment:
Shouldn't we also remove this?

Reply:
We keep that because it’s the oldest version of Kubernetes where Submariner still works. Whether that is worth keeping is debatable of course!